src/components/timeline/TimelinePanel.svelte

```diff
@@ -84,21 +88,32 @@
 
   function onMouseDown(event: CustomEvent<MouseDown>) {
     const { detail } = event;
-    const { activityDirectives, spans } = detail;
-    if (spans.length) {
+    const { activityDirectives, spans, externalEvents } = detail;
+    if (externalEvents != null && externalEvents.length) {
```

Comment:
Prefer `!==` which will also force us to be explicit about null vs undefined in the detail